src/locales/cy/translation.json

```diff
@@ -196,14 +196,14 @@
       }
     },
     "pyiF2fTechnical": {
-      "title": "Mae'n ddrwg gennym, ni allwn brofi pwy ydych chi",
```

Comment:
Ideally we’d use typographically correct apostrophes rather than tick marks here - type option+shift+] for ’ and option+[ for ‘ – see https://smartquotesforsmartpeople.com/

Reply:
So the changes here are correct?

Reply:
yes, all good. Thank you!